util/session/sessionmanager.go

```diff
@@ -141,9 +145,10 @@ func (mgr *SessionManager) Parse(tokenString string) (jwt.Claims, error) {
 		return nil, err
 	}
 
-	if id := jwtutil.GetField(claims, "jti"); id != "" && account.TokenIndex(id) == -1 {
```

Comment:
This check is still required. Argo CD issues two types of tokens: account and project token. Project tokens can be distinguished by subject (`rbacpolicy.IsProjectSubject(subject)` ) . 

Both project and account tokens might be revoked by the user. The `SessionManager.Parse` must ensure that token is not revoked. Removed verification ensures that account token is not removed. 

Project token verification is implemented here:
https://github.com/argoproj/argo-cd/blob/3c2be618276369c0a9e60a49d9b8151f0136fde7/server/rbacpolicy/rbacpolicy.go#L89-L95

I suggest to move it into `SessionManager.Parse` and update to take into account that project token has id as well. We need to support legacy project tokens without id as well.